pkg/runtime/executor.go

```diff
@@ -27,22 +27,23 @@ import (
 
 // Executor object
 type Executor struct {
-	filePath     string
-	dirPath      string
-	policyPath   []string
-	cloudType    []string
-	iacType      string
-	iacVersion   string
-	configFile   string
-	scanRules    []string
-	skipRules    []string
-	iacProvider  iacProvider.IacProvider
-	policyEngine []policy.Engine
-	notifiers    []notifications.Notifier
+	filePath      string
+	dirPath       string
+	policyPath    []string
+	cloudType     []string
+	iacType       string
+	iacVersion    string
+	configFile    string
+	scanRules     []string
+	skipRules     []string
+	iacProvider   iacProvider.IacProvider
+	policyEngines []policy.Engine
+	notifiers     []notifications.Notifier
+	severity      string
 }
 
 // NewExecutor creates a runtime object
-func NewExecutor(iacType, iacVersion string, cloudType []string, filePath, dirPath, configFile string, policyPath, scanRules, skipRules []string) (e *Executor, err error) {
+func NewExecutor(iacType, iacVersion string, cloudType []string, filePath, dirPath, configFile string, policyPath, scanRules, skipRules []string, severity string) (e *Executor, err error) {
```

Comment:
The function input vars are increasing, we may need to do something about this. Not necessarily in this PR :)